.perltidyrc

```diff
@@ -0,0 +1,17 @@
+# Workaround needed for handling non-ASCII in files.
```

Comment:
Can you make a PR for os-autoinst and openQA with this config (and the applied changes)? Then we would see what changes it would make.

Reply:
193 files changed in openQA & 84 files in os-autoinst... WOAH :joy:

Reply:
These two are the tests:

- https://github.com/os-autoinst/openQA/pull/5348
- https://github.com/os-autoinst/os-autoinst/pull/2399